### PR TITLE
Fixes #2405 mapping lost after configure

### LINKED ISSUE #2405: 11.1 Observed data mapping in a simulation getting resetted during Clone or Configure
Original simulation:
grafik

Cloned Simulation:
grafik

### PR DESCRIPTION
**Pull request by msevestre**

### CODE REVIEW

File: src/PKSim.Core/Model/IndividualSimulation.cs [hunk 1]
```diff
@@ -40,25 +37,7 @@ public virtual DataRepository DataRepository
 
       public int IndividualId => Results.AllIndividualIds().FirstOrDefault();
 
-      public override TBuildingBlock BuildingBlock<TBuildingBlock>()
-      {
-         return AllBuildingBlocks<TBuildingBlock>().SingleOrDefault();
-      }
-
-      public override void UpdatePropertiesFrom(IUpdatable sourceObject, ICloneManager cloneManager)
```
Comment (msevestre) [inline src/PKSim.Core/Model/IndividualSimulation.cs:48]: I am not sure why those guys were defined an empty. Probably after some refactoring. Removed

File: src/PKSim.Presentation/Presenters/Simulations/ConfigureSimulationPresenter.cs [hunk 2]
```diff
@@ -152,7 +160,12 @@ protected override void UpdateSimulationProperties()
 
          _simulationModelCreator.CreateModelFor(Simulation);
 
-         //now update all parameters from the orginal simulationToClone
+         //After the simulation was created, we can update the output mapping (we need the model to be available in the simulation)
+         //to make sure we swap out the simulation references
+         _originalSimulation.OutputMappings.Each(x => Simulation.OutputMappings.Add(x.Clone()));
```
Comment (msevestre) [inline src/PKSim.Presentation/Presenters/Simulations/ConfigureSimulationPresenter.cs:165]: This is the code change here. When we change individual, or compound, we recreate a whole simulation completely. This clones the output and we have to swap the reference as well

Comment (msevestre) [reply src/PKSim.Presentation/Presenters/Simulations/ConfigureSimulationPresenter.cs:165]: it's also working as expected when the compound is swapped. The path is still pointing to the old one, but it's marked as invalid and is shown as None in the UI
@georgeDaskalakis Do you see any problem with this?

File: tests/PKSim.Tests/Presentation/ConfigureSimulationPresenterSpecs.cs [hunk 3]
```diff
@@ -113,6 +128,13 @@ public void should_update_the_parameters_from_the_original_simulation()
       {
          A.CallTo(() => _simulationParametersUpdater.ReconciliateSimulationParametersBetween(_originalSimulation, _clonedSimulation, PKSimBuildingBlockType.Simulation)).MustHaveHappened();
       }
+
+      [Observation]
+      public void should_have_updated_the_output_mapping_based_on_the_original_simulation()
+      {
+         _clonedSimulation.OutputMappings.Count().ShouldBeEqualTo(1);
```
Comment (msevestre) [inline tests/PKSim.Tests/Presentation/ConfigureSimulationPresenterSpecs.cs:135]: the test

**Review by Yuri05 — APPROVED** [review]

**Review by rwmcintosh — APPROVED** [review]

**Review by georgeDaskalakis — APPROVED** [review]

**Pull request merged**